src/main/java/tech/blacksource/blacknectar/service/stores/Store.java

```diff
@@ -248,12 +258,13 @@ private JsonObject createJSON()
         jsonObject.addProperty(Keys.NAME, this.name);
         jsonObject.add(Keys.LOCATION, location.asJSON());
         jsonObject.add(Keys.ADDRESS, this.address.asJSON());
+        jsonObject.addProperty(Keys.IS_FARMERS_MARKET, isFarmersMarket);
```

Comment:
Looks good; straightforward. Add a test that checks the json contains the right variable.